e2e/client/playwright/publish-queue.spec.ts

```diff
@@ -26,3 +27,31 @@ test.skip('item appearing in publish queue after publishing', async ({page}) =>
 
     await expect(page.locator(s('publish-queue-item=test sports story'))).toBeAttached({timeout: 10000});
 });
+
+test('item appearing in publish queue after publishing with subscriber', async ({page}) => {
```

Comment:
What are you attempting to test here? This test looks the same as the one above. If subscribers dropdown was broken it would probably still pass? If so, we're not testing anything that's not being tested already.